svcs/data/caddy/Caddyfile.blue

```diff
@@ -0,0 +1,44 @@
+# The Caddyfile is an easy way to configure your Caddy web server.
+#
+# Unless the file starts with a global options block, the first
+# uncommented line is always the address of your site.
+#
+# To use your own domain name (with automatic HTTPS), first make
+# sure your domain's A/AAAA DNS records are properly pointed to
+# this machine's public IP, then replace ":80" below with your
+# domain name.
+
+#:80 {
+    # Set this path to your site's directory.
+#	root * /usr/share/caddy
+
+    # Enable the static file server.
+#	file_server
```

Comment:
We can get rid of this header comment and replace it with a note about where to put/how to use this file, what it sets up, and how this is different from the other Caddyfile.